### PR TITLE
Ayr 839/browse tb all views

### PR DESCRIPTION
**Pull request by colinbowen**
## Changes in this PR
- Add updated browse all, browse transferring body mobile views

## JIRA ticket
https://national-archives.atlassian.net/browse/AYR-839

### After
Browse All
Screenshot 2024-03-21 at 18 20 18

Browse Transferring Body
Screenshot 2024-03-21 at 18 20 41

- [ ] Requires env variable(s) to be updated

### CODE REVIEW

**Comment by pgandhizaizi** [comment]
Please e2e tests are working as expected, if not please get them fixed

File: e2e_tests/test_browse.py [hunk 1]
```diff
@@ -55,14 +65,18 @@ def test_browse_filter_functionality_with_query_string_parameters(
         header_rows = aau_user_page.locator(
             "#tbl_result tr:visible"
         ).evaluate_all(
-            """els => els.slice(0).map(el =>
-              [...el.querySelectorAll('th')].map(e => e.textContent.trim())
-            )"""
+            """els => els.map(el => {
+                var headerCells = [...el.querySelectorAll('th')];
+                headerCells.shift();
+                return headerCells.map(e => e.textContent.trim());
```
Comment (pgandhizaizi) [inline e2e_tests/test_browse.py]: why headerCelss.shift() has been used, can you not use the class to identify a row similar to the implementation in e2e_tests/test_search.py

rows = aau_user_page.locator("#tbl_result tr.top-row").evaluate_all(
            """els => els.slice(0).map(el =>
              [...el.querySelectorAll('td.search__mobile-table__top-row')].map(e => e.textContent.trim())
            )"""
        )

File: e2e_tests/test_browse_transferring_body.py [hunk 2]
```diff
@@ -130,16 +142,20 @@ def test_browse_transferring_body_sort_functionality_by_series_descending(
         header_rows = standard_user_page.locator(
             "#tbl_result tr:visible"
         ).evaluate_all(
-            """els => els.slice(0).map(el =>
-              [...el.querySelectorAll('th')].map(e => e.textContent.trim())
-            )"""
+            """els => els.map(el => {
+                var headerCells = [...el.querySelectorAll('th')];
+                if (headerCells.length >= 3) {
+                    headerCells.splice(2, 1);
```
Comment (pgandhizaizi) [inline e2e_tests/test_browse_transferring_body.py:136]: why headerCelss.shift() has been used, can you not use the class to identify a row similar to the implementation in e2e_tests/test_search.py

rows = aau_user_page.locator("#tbl_result tr.top-row").evaluate_all(
            """els => els.slice(0).map(el =>
              [...el.querySelectorAll('td.search__mobile-table__top-row')].map(e => e.textContent.trim())
            )"""
        )

Comment (colinbowen) [reply e2e_tests/test_browse_transferring_body.py:136]: .shift has been removed

File: e2e_tests/test_browse.py (hunk 1 above)
Comment (colinbowen) [reply e2e_tests/test_browse.py]: .shift has been removed.

File: e2e_tests/test_browse.py [hunk 3]
```diff
@@ -45,25 +55,23 @@ def test_browse_filter_functionality_with_query_string_parameters(
...
+              [...el.querySelectorAll('td.browse__table__all_desktop')].map(e => e.textContent.trim())
             )"""
         )
 
         expected_rows = [
-            ["Mock 1 Department", "MOCK1 123", "05/03/2024", "83", "11"],
-            ["Testing A", "TSTA 1", "25/01/2024", "63", "5"],
+            ["MOCK1 Department", "MOCK1 123", "28/07/2023", "1", "1"],
+            ["Testing A", "TSTA 1", "25/01/2024", "73", "7"],
         ]
```
Comment (pgandhizaizi) [inline e2e_tests/test_browse.py:72]: the test values are validated against 'ayr' db , so please do not change the expected_rows values , as they already passed the tests based on the data in 'ayr' db

Comment (pgandhizaizi) [reply e2e_tests/test_browse.py:72]: this apply to all the test cases

File: e2e_tests/test_browse.py (hunk 1 above)
Comment (colinbowen) [reply e2e_tests/test_browse.py]: resolved on call

File: e2e_tests/test_browse_transferring_body.py (hunk 2 above)
Comment (colinbowen) [reply e2e_tests/test_browse_transferring_body.py:136]: resolved on call

File: e2e_tests/test_browse.py (hunk 3 above)
Comment (colinbowen) [reply e2e_tests/test_browse.py:72]: resolved on call, test cases now using ayr db

**Review by pgandhizaizi — APPROVED** [review]
looks good

**Pull request merged**